packages/flutter_tools/lib/src/build_runner/build_runner.dart

```diff
@@ -121,6 +123,9 @@ class BuildRunner {
     } finally {
       status.stop();
     }
+    if (disableKernelGeneration) {
+      return const BuildResult(null, null);
```

Comment:
If we return nothing in the `BuildResult` then what _do_ we build when kernel generation is disabled? 🤔 Maybe leave a comment?